binding.cc

```diff
@@ -1107,6 +1197,77 @@ namespace zmq {
     BufferReference* bufref_;
   };
 
+  NAN_METHOD(Socket::Sendv) {
+    Socket* socket = GetSocket(info);
+    if (socket->state_ != STATE_READY)
+      return info.GetReturnValue().Set(Nan::False());
```

Comment:
There are primitive setters for integers as well: https://github.com/nodejs/nan/blob/master/doc/methods.md#api_nan_return_value